MSFragger-GUI/src/com/dmtavt/fragpipe/tools/percolator/PercolatorOutputToPepXML.java

```diff
@@ -173,6 +173,12 @@ private static StringBuilder handle_search_hit(final List<String> searchHit, fin
         if (!Float.isNaN(nttNmc.RTscore)) {
             sb.append(String.format("<search_score name=\"rtscore\" value=\"%f\"/>\n", nttNmc.RTscore));
         }
+        if (!Float.isNaN(nttNmc.ionMobility)) {
+            sb.append(String.format("<search_score name=\"ionmobility\" value=\"%f\"/>\n", nttNmc.ionMobility));
+        }
```

Comment:
I think we already have ion mobility in the pepxml file.

Reply:
We do, sending another commit